service/src/main/java/io/camunda/service/VariableServices.java

```diff
@@ -61,4 +66,22 @@ public VariableEntity getByKey(final Long key) {
                             VARIABLE_READ_AUTHORIZATION, VariableEntity::processDefinitionId)))
                 .getVariable(key));
   }
+
+  public CompletableFuture<VariableRecord> createClusterVariable(
+      final String name, final Object value) {
+    return sendBrokerRequest(
+        new BrokerCreateVariableRequest()
+            .clusterLevel()
+            .setVariable(name, getDocumentOrEmpty(value)));
```

Comment:
❓ What happens when value is `null` or `""` (an empty string). Do we represent it as an empty document `{}`? Let's make sure we have a test that shows we can create a variable with `null` as value (end-to-end to ensure we cover both this and the engine).